files/jcl/ZWENOSEC.jcl

```diff
@@ -203,6 +204,17 @@
   LISTGRP  &ADMINGRP. OMVS
   DELGROUP &ADMINGRP.
 
+/* REMOVE ZOWE RESOURCE CLASS ...................................... */
+/* uncomment commands to below if the ZOWE class has been created    */
+/* by the ZWESECUR job                                               */
+
+/* remove profile in ZOWE class                                      */
+/* RDELETE ZOWE APIML.SERVICES                                       */
+/* remove ZOWE class                                                 */
```

Comment:
We worked with the RACF folks, and ZOWE will be a standard SAF class provided by z/OS, starting at v2.5, so there there is no deleting it anymore. Like with TSS, add a bigger comment to this sample command
```
/* uncomment commands to below if the ZOWE class has been created    */
/* by the ZWESECUR job                                               */
```

Reply:
Hi Onno, thank you for the review. All three ESMs have these commands commented and preceded with an explanatory comment now.